### DISCUSSION TITLE
App update

### DISCUSSION THREAD

**Discussion by owenconnorz** [z-huang/music#314] [General]
Is the update meant to be on the auto updater in the app now as its not showing

**Comment by z-huang** [comment]
In-app updater is removed in 0.4.1. It's recommended to use F-Droid or Droidify to do update.

**Reply by owenconnorz** [reply]
Ahh that's a shame no worries

**Comment by owenconnorz** [comment]
By any chance can I upload the apks to aptoid as f droid and droidify doesn't work well for me  and add the changelogs to it for u no worries if not it just works better for me and also people I know

**Reply by z-huang** [reply]
After googling, it seems that user can upload apps to Aptoide as long as they want to. So, go ahead for your convenience.

**Reply by owenconnorz** [reply]
Thanks so much will help friends out and gain u some more users 😁
